addons/frag/functions/fnc_dev_debugAmmo.sqf

```diff
@@ -17,65 +21,89 @@
  */
 
 params [
-    ["_debugMissing", true, [false]],
-    ["_debugForce", false, [false]],
-    ["_debugSkippedFragPower", 30, [0]]
+    ["_logAll", false, [false]],
+    ["_printOnlyIncomplete", true, [true]],
+    ["_onlyShotAmmoTypes", false, [false]],
+    ["_csvFormat", false, [false]]
 ];
 
 diag_log text format ["~~~~~~~~~~~~~Start [%1]~~~~~~~~~~~~~", _this];
+if (_csvFormat) then {
+    diag_log text format ["ammo,gurney_c,gurney_m,gurney_k,gurney_gC,skip,Inheritance"];
+};
 
-private _allMagsConfigs = configProperties [configFile >> "CfgMagazines", "isClass _x", true];
-private _processedCfgAmmos = [];
+// Gather all configs, either those that could be created from firing or all classes
+private _allAmmoConfigs = [];
```

Comment:
I updated the function to do what I think it should for the current version of frag and didn't try to hard to be super efficient about it since I was able to run it with RHS/CUP/multiple CDLC loaded in a couple hundred milliseconds. I can definitely do that if I have time tomorrow.

Reply:
Did some optimization in https://github.com/acemod/ACE3/pull/10157/commits/185544160af7a5da6300c851c2fe6b34c59c1b55. I don't think there are any other similar searches in the code, and couldn't find any, but please highlight any you see.